.npmignore

```diff
@@ -1,7 +0,0 @@
-.vs
```

Comment:
Why do we not want to ignore this stuff?

Reply:
The `files` attribute in `package.json` tells npm to only include the `dist` directory.